aws/resource_aws_elasticsearch_domain.go

```diff
@@ -194,6 +194,10 @@ func resourceAwsElasticSearchDomain() *schema.Resource {
 							Type:     schema.TypeBool,
 							Optional: true,
 						},
+						"zone_awareness_count": {
```

Comment:
Two notes:
* This argument needs documentation added in `website/docs/r/elasticsearch_domain.html.markdown`
* Since this argument is not marked `ForceNew: true`, an additional `TestStep` should be added to the acceptance test that attempts to update or remove it